addons/antrea/0.13.1/Manifest

```diff
@@ -0,0 +1 @@
+image antrea-ubuntu projects.registry.vmware.com/antrea/antrea-ubuntu:v0.13.1
```

Comment:
this scares me
does it work on non-ubuntu OSes?

Reply:
Yes, works on CentOS